src/stylesheets/settings/_settings-spacing.scss

```diff
@@ -121,3 +121,30 @@ $theme-nav-width:           'desktop' !default; // was 951px
 $theme-site-max-width:      'desktop' !default; // was 1040px
 
 $sidenav-current-border-width:  0.5 !default; // must be in rem for math
+
+$text-max-width:                66ch !default; // 66 characters per line, TODO: use token?
+$text-wide-width:               72ch !default; // TODO: use token?
+$lead-max-width:                77rem !default; // TODO: use token?
+$site-max-width:                $theme-site-max-width !default; // TODO: remove variable and occurances from code
+$site-margins:                  $theme-site-margins !default;
+$site-margins-mobile:           $theme-site-margins-mobile !default;
+$input-max-width:               46rem !default; // TODO: use token?
+$checkbox-border-radius:        units(2px) !default;
+$focus-spacing:                 0 !default;
+$nav-width:                     951px !default; // TODO: remove variable and occurances from code
+$search-min-width:              27ch; // minimum length of 27 letters for typical search query - magic number
+
+// 44 x 44 pixels hit target following Apple iOS Human Interface Guidelines
+$hit-area: 4.4rem !default;
+
```

Comment:
Why do we have two groups of spacing variables? Are they both being used? Should there be only one? How confident are we of the chosen ones at these increments?